yogstation/code/modules/clothing/suits/wiz_robe.dm

```diff
@@ -0,0 +1,26 @@
+/obj/item/clothing/head/wizard/red/fake
+	name = "red wizard hat"
+	desc = "Strange-looking red hat-wear that most certainly belongs to a real magic user."
+	icon_state = "redwizard"
+	gas_transfer_coefficient = 1
+	permeability_coefficient = 1
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	resistance_flags = FLAMMABLE
+
+/obj/item/clothing/head/wizard/yellow/fake
```

Comment:
Just use ``/obj/item/clothing/head/wizard/fake`` and change the ``icon_state`` instead of copying all of this 3 times

Reply:
I could've done that but I wanted to follow the naming scheme already established

Reply:
Naming scheme doesn't matter, this ensures the wizard hate will always be fake incase /tg/ adds something.